### PR TITLE
Camel 14781 security page breadcrumps

### PR DESCRIPTION
**Pull request by Siyubu**
While navigating on apache website I saw pages that don't have toolbars breadcrumps such as security page from About/ security.
Problem resolution:
From the root repository under layouts/ partials/seo/structured I built a breadcrumbs.html partial which get and display the current page user is on.
The partial has been included in root/security/list.html unde main div.
The purpose of making breadcamps.html a partial is making easy the act of displaying breadcrumps on any camel-website page. what is needed is to include it in the targeted page.

### CODE REVIEW

**Comment by Siyubu** [comment]
This pull request containt antora_ui_camel/yarn.lock as removing it on CAMEL_14781_security_page_breadcrumps branch level is resulting to antora_ui_camel/yarn.lock merge conflict

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -53,6 +53,10 @@
     <script src="{{ path.Join "_" (index .Site.Data "rev-manifest" "js/site.js") | relURL }}"></script>
     <script async src="{{ path.Join "_" (index .Site.Data "rev-manifest" "js/vendor/highlight.js") | relURL }}"></script>
     <script async src="{{ path.Join "_" (index .Site.Data "rev-manifest" "js/vendor/svg4everybody.js") | relURL }}"></script>
+
```
Comment (Delawen) [inline layouts/partials/footer.html:56]: Are these lines needed?

File: layouts/partials/header.html [hunk 2]
```diff
@@ -27,6 +27,7 @@
     <meta name="keywords" content="{{ .Keywords }}">
     {{ end }}
     <link rel="stylesheet" href="{{ path.Join "_" (index .Site.Data "rev-manifest" "css/site.css") | relURL }}">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```
Comment (Delawen) [inline layouts/partials/header.html:30]: We shouldn't use a library hosted by an external entity. Specially if that external entity is known for invasively tracking users.

Either use other existing library or add it to our dependencies and have it on our host.

File: layouts/security/list.html [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 <div class="body">
     <main role="main">
+        {{ partial "seo/structured/breadcrumbs.html" . }}
         <article class="static doc">
-
```
Comment (Delawen) [inline layouts/security/list.html:6]: Usually, don't add or remove empty lines unless it is really needed. It makes the review process simpler if we only see what is really changing something.

**Comment by zregvart** [comment]
@Siyubu are you still working on this, should we close this?

**Comment by Siyubu** [comment]
You may close it

**Pull request closed without merging**